docs/architecture/grpc-for-wcf-developers/kubernetes.md

```diff
@@ -33,39 +33,39 @@ metadata:
+Pods are the basic units of execution in Kubernetes. They can run multiple containers, but they're also used to run single containers. The pod also includes any storage resources required by the containers, and the network IP address.
 
 ### Services
 
-Services are meta-objects that describe pods (or sets of pods) and provide a way to access them within the cluster, such as mapping a service name to a set of pod IP addresses using the cluster DNS service.
+Services are meta-objects that describe Pods (or sets of Pods) and provide a way to access them within the cluster, such as mapping a service name to a set of pod IP addresses by using the cluster DNS service.
 
 ### Deployments
 
-Deployments are the *described state* objects for Pods. If you create a Pod manually, when it terminates it won't be restarted. Deployments are used to tell the cluster which pods, and how many replicas of those pods, should be running at the present time.
+Deployments are the *described state* objects for Pods. If you create a pod manually, it won't be restarted when it terminates. Deployments are used to tell the cluster which Pods, and how many replicas of those Pods, should be running at the present time.
 
 ### Other objects
 
-Pods, Services, and Deployments are just three of the most basic object types. There are dozens of other types of object that are managed by a Kubernetes cluster. For more information, see the [Kubernetes Concepts](https://kubernetes.io/docs/concepts/) documentation.
+Pods, Services, and Deployments are just three of the most basic object types. There are dozens of other object types that are managed by Kubernetes clusters. For more information, see the [Kubernetes Concepts](https://kubernetes.io/docs/concepts/) documentation.
 
 ### Namespaces
 
-Kubernetes clusters are designed to scale to hundreds or thousands of nodes, and run similar numbers of services. To avoid clashes between object names, namespaces are used to group objects together as part of larger applications. Kubernetes own services run in a `default` namespace. All user objects should be created in their own namespaces to avoid potential clashes with default objects or other tenants in the cluster.
+Kubernetes clusters are designed to scale to hundreds or thousands of nodes and to run similar numbers of services. To avoid clashes between object names, namespaces are used to group objects together as part of larger applications. Kubernetes's own services run in a `default` namespace. All user objects should be created in their own namespaces to avoid potential clashes with default objects or other tenants in the cluster.
 
 ## Get started with Kubernetes
 
-If you're running Docker Desktop for Windows or macOS, Kubernetes is already available. Just enable it in the Kubernetes section of the Settings window.
+If you're running Docker Desktop for Windows and macOS, Kubernetes is already available. Just enable it in the **Kubernetes** section of the **Settings** window:
```

Comment:
```suggestion
If you're running Docker Desktop for Windows or Docker Desktop for Mac, Kubernetes is already available. Just enable it in the **Kubernetes** section of the **Settings** window:
```

The word `and` changed the meaning here. To make it explicit I used the verbatim names of the products